transmogrifier/sources/transformer.py

```diff
@@ -68,6 +69,9 @@ def __next__(self) -> timdex.TimdexRecord:
             except DeletedRecordEvent as error:
                 self.deleted_records.append(error.timdex_record_id)
                 continue
+            except SkippedRecordEvent:
+                self.skipped_record_count += 1
+                continue
```

Comment:
I'm unable to comment on the lines below, as they did not change, but thoughts on updating the logic to treat a successful record as the default path?  

I know this will get touched on in orchestration updates, but while touching this `__next__` method, thought it could be a good time to nudge it.

e.g.
```python
if not record:
    self.skipped_record_count += 1
    continue
self.transformed_record_count += 1
return record  
```

Reply:
Good call on the exception message and I was frustrated with the repetition in that method so thank you very much for such a simple refactor!